mopidy/mpd/actor.py

```diff
@@ -57,9 +60,7 @@ def on_stop(self):
         process.stop_actors_by_class(session.MpdSession)
 
```

Comment:
I'm considering the following change to this handling as we don't really ever care about the `kwargs` we just need to know what event fired and map it. Thoughts?

``` 
_CORE_EVENTS_TO_IDLE_SUBSYSTEMS = {                                             
    'track_playback_paused': None,                                              
    'track_playback_resumed': None,                                             
    'track_playback_started': None,                                             
    'track_playback_ended': None,                                               
    'playback_state_changed': 'player',                                         
    'tracklist_changed': 'playlist',                                            
    'playlists_loaded': 'stored_playlist',                                      
    'playlist_changed': 'stored_playlist',                                      
    'playlist_deleted': 'stored_playlist',                                      
    'options_changed': 'options',                                               
    'volume_changed': 'mixer',                                                  
    'mute_changed': 'output',                                                   
    'seeked': 'player',                                                         
    'stream_title_changed': 'playlist',                                         
} 

....

class MpdFrontend(...):
    ....

   def on_event(self, event, **kwargs):                                          
        if event not in _CORE_EVENTS_TO_IDLE_SUBSYSTEMS:                          
            logger.warning(                                                       
                'Got unexpected event: %s(%s)', event, ', '.join(kwargs))         
        else:                                                                     
            self.send_idle(_CORE_EVENTS_TO_IDLE_SUBSYSTEMS[event]) 
```